src/EditorFeatures/Core.Wpf/InlineRename/InlineRenameSession.cs

```diff
@@ -69,6 +71,12 @@ private set
             }
         }
 
+        /// <summary>
+        /// Information about whether a file rename should be allowed as part
+        /// of the rename operation, as determined by the language
+        /// </summary>
+        public InlineRenameFileRenameInfo FileRenameInfo { get; }
```

Comment:
FWIW, i think FileRenameInfo is a pefectly fine name for the type as well.

Reply:
Eh, I already typed all this out :)